packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -137,6 +137,13 @@ export interface MutualAuthentication {
    * @default false
    */
   readonly ignoreClientCertificateExpiry?: boolean;
+
+  /**
+   * Indicates whether trust store CA names are advertised
+   *
+   * @default AdvertiseTrustStoreCaNames.OFF
+   */
+  readonly advertiseTrustStoreCaNames?: AdvertiseTrustStoreCaNames;
```

Comment:
What about changing the type to a `boolean` and removing the `enum`? It would provide a cleaner interface

Reply:
Okay I will try it

Reply:
Thanks 👍 You can find a sample for the suggested approach [here](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-certificatemanager/lib/certificate.ts#L94)

Reply:
Thank you for the nice sample! I made an update according to it.